src/main/target/FLYWOOF7DUAL/target.h

```diff
@@ -20,7 +20,7 @@
 
 #pragma once
 
-#define USE_TARGET_CONFIG					 
+#define USE_TARGET_CONFIG                    
```

Comment:
👾

Reply:
When I commit I run a script to convert tabs to spaces. Ideally there would be no trailing spaces either, but they're better than tabs.

Reply:
Fixed